README.md

```diff
@@ -129,6 +129,26 @@ stackit config list
 
 You can also edit the configuration file manually.
 
+## Customization

```

Comment:
Re-structure this section to be more clear like this:
```
## Customization

### Pager

To specify a custom pager, use the `PAGER` environment variable.

If the variable is not set, STACKIT CLI uses the `less` as default pager.
When using `less` as a pager .....

[IMPORTANT] 
....
```

Reply:
Updated the section structure as suggestion